_posts/2023-01-09-knn-similarity-refresh.markdown

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title:  "Build k-Nearest Neighbor (k-NN) Similarity Search Engine with OpenSearch"
+authors:
+- vamshin
+- laijiang
+- csw
+- jmazane
+date: 2023-01-09 00:00:00 -0700
```

Comment:
I don't believe we typically update the date here - please confirm @pajuric - and please set this back to the original blog date. This isn't a new blog, it's a refresh.

Reply:
Correct - we leave the original publish date for the blog, which was Mon, Apr 06, 2020.

Reply:
Put the original date, but named the blog today's date.